### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by LinhWuerzburger**
- replacing control function with pseudo unit tests with real unit tests.

### CODE REVIEW

File: gtests/domain/Multigrid.cpp [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    // domain inner cells
+    EXPECT_EQ(0, multigrid.get_start_index_domain_inner_cells_level_joined(0));
+    EXPECT_EQ(383, multigrid.get_end_index_domain_inner_cells_level_joined(0));
+
+    EXPECT_EQ(384, multigrid.get_start_index_domain_inner_cells_level_joined(1));
+    EXPECT_EQ(431, multigrid.get_end_index_domain_inner_cells_level_joined(1));
+
+    EXPECT_EQ(432, multigrid.get_start_index_domain_inner_cells_level_joined(2));
+    EXPECT_EQ(437, multigrid.get_end_index_domain_inner_cells_level_joined(2));
+}
```
Comment (ChristianVonMach) [inline gtests/domain/Multigrid.cpp:97]: missing newline

File: gtests/utility/Mapping.cpp [hunk 2]
```diff
@@ -108,3 +108,21 @@ TEST(MappingTest, getFieldTypeName) {
...
+}
+
+TEST(MappingTest, toPatch) {
+    EXPECT_EQ(Patch::LEFT, Mapping::to_patch(CoordinateAxis::X, true));
+    EXPECT_EQ(Patch::RIGHT, Mapping::to_patch(CoordinateAxis::X, false));
+    EXPECT_EQ(Patch::BOTTOM, Mapping::to_patch(CoordinateAxis::Y, true));
+    EXPECT_EQ(Patch::TOP, Mapping::to_patch(CoordinateAxis::Y, false));
+    EXPECT_EQ(Patch::FRONT, Mapping::to_patch(CoordinateAxis::Z, true));
+    EXPECT_EQ(Patch::BACK, Mapping::to_patch(CoordinateAxis::Z, false));
+}
```
Comment (ChristianVonMach) [inline gtests/utility/Mapping.cpp:128]: missing newline

File: gtests/domain/Multigrid.cpp (hunk 1 above)
Comment (LinhWuerzburger) [reply gtests/domain/Multigrid.cpp]: 8e8e7ef

File: gtests/utility/Mapping.cpp (hunk 2 above)
Comment (LinhWuerzburger) [reply gtests/utility/Mapping.cpp]: fd95a9a

**Review by ChristianVonMach — DISMISSED** [review]

**Review by ChristianVonMach — APPROVED** [review]

**Pull request merged**